tests/kolainst/destructive/soft-reboot.sh

```diff
@@ -76,6 +76,11 @@ case "${AUTOPKGTEST_REBOOT_MARK:-}" in
   # Verify that soft-reboot state files are gone
   test '!' -f /run/ostree/nextroot-booted
   
+  # And verify there were no failures to unmount /etc or /sysroot
+  journalctl -b -1 -u sysroot.mount -u etc.mount >previous-journal.txt
```

Comment:
When you do a soft reboot, for the journal it's still the same boot, ie you must use `-b0` here

Reply:
Thinking about it more, the journal is pretty small, we can just remove the `-b`